tests/app/celery/test_tasks.py

```diff
@@ -970,79 +889,28 @@ def test_should_not_save_sms_if_restricted_service_and_invalid_number(self, noti
         job = create_job(template)
         notification = _notification_json(template, to="07700 900849")
 
-        save_sms_mock = mocker.patch("app.celery.tasks.save_sms")
-
-        process_row(
-            Row(
-                {"foo": "bar", "to": notification["to"]},
-                index="row_num",
-                error_fn=lambda k, v: None,
-                recipient_column_headers=["to"],
-                placeholders={"foo"},
-                template=SMSMessageTemplate(template.__dict__),
-            ),
-            template,
-            job,
-            service,
-        )
-
-        assert not save_sms_mock.called
-
-    def test_should_not_save_sms_if_team_key_and_recipient_not_in_team(self, notify_db_session, mocker):
```

Comment:
Do we have another test to replace this one?

Reply:
Hmm these tests in the PR seems to test the happy path, but that one removed is for a SMS that shouldn't be sent, right?

Reply:
True! I can make a note to add this test in another PR (as this code path wasn't being used anyways with the feature flag turned on ). Does that work?

Reply:
Yeah, as long as we track it the task card. It's not a blocker to this PR.

Reply:
added https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/774